build/link-wrapper.sh

```diff
@@ -32,6 +35,12 @@ case $1 in
 		    $CC $OPTIONS
 		fi
 		;;
+		freebsd)
+		if [ -z ${CC+x} ] && type -p clang >/dev/null; then
+			CC=clang++
+		fi
```

Comment:
That was the last one that needed to be eliminated- if it works without it finally I'm glad it's not a problem anymore yay!

Reply:
@alexmiller-apple I looked at this issue again I'm with you none of this needs to be in the bash scripts. Looking at various other FreeBSD projects developers are instructed to export CC="/usr/bin/clang" or write that setting into one of the various places in their environment. What I've been using the whole time is: FreeBSD-11.1-RELEASE-amd64-bootonly.iso
It would be better along with the notice about production also put a link to a file called "Setting up your FreeBSD build environment" with some basic info about making sure this is set and to check that it is before proceeding with installing the necessary packages. If I got confused about it surely somebody else could too. Running gmake is not the norm generally also the BSD ports system handles these things behind the scenes. In that file information about which FreeBSD iso release is used would also help. What image are you using to test this? I'd like to update to 11.2 and use the standard release this time.